src/config/laravel-query-logger.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+return [
+
+	'enabled' => true,
+
+    /*
+     * Min size of query to log in seconds
+     */
+
+    'min_size' => 0,
```

Comment:
I think the name `min_size` is not clearable.

Reply:
Maybe milliseconds is more useful?

Reply:
What name do you think its better? maybe `minimum_size_in_miliseconds`?